_config.yml

```diff
@@ -1,21 +1,44 @@
 # Site settings
 title: Pyronear
 description: "Détection précoce de feux de forêt" # this means to ignore newlines until "baseurl:"
```

Comment:
this com is useless now
```suggestion
description: "Détection précoce de feux de forêt"
```

Reply:
Thanks, I'll remove comments in another PR to avoid requesting reviews :ok_hand: